src/library/trackcollectionmanager.h

```diff
@@ -13,7 +13,7 @@
 class TrackCollection;
 class ExternalTrackCollection;
 
-// Manages the internal and a varying number of external track collections.
+// Manages Mixxx's internal database of tracks as well as external track collections.
```

Comment:
Ugh, GitHub made a commit with a useless commit description :( Can you use `git commit --amend` and force push with a better commit description?

Reply:
Doesn't matter